attach-remotes

```diff
@@ -10,7 +10,7 @@ Jenkins.instance.nodes.each {
 
 
 
-String[] agentList = ["quigon", "anakin", "yoda", "obiwan"]
+String[] agentList = ["ci-james-01", "ci-james-02", "ci-james-03", "ci-james-04"]
```

Comment:
we only have 4 ? It thought we decided 2 for each team member

Reply:
the 2 other are reserved for future use (load testing) for now